verbit/streaming_client.py

```diff
@@ -263,12 +274,13 @@ def _connect_websocket(self, media_config: MediaConfig, response_types: Response
         Which is linked and documented in tenacity:
         https://tenacity.readthedocs.io/en/latest/api.html#wait-functions
 
+        :param ws_url: base ws url to use
```

Comment:
why base? it's the same url passed all along the way, no?

Reply:
you are right, done